docs/Publish.md

```diff
@@ -0,0 +1,83 @@
+# Publish & Deploy Apps
```

Comment:
I recommend moving from `docs/Publish.md` to `docs/Running/Publish.md` similar to what is in [this commit](https://github.com/enarx/enarx/pull/2059/commits/4bd83528bf22f21e26cd8ca3bc7e9f7be771c29c). I will then remove `Publish.md` from #2059 

@rvolosatovs put in some comments in #2059 around `Publish.md` which are worth looking into.

Reply:
I've moved it to Running/Publish.md. I've also responded to all of Roman's comments in #2059 and incorporated several of them. I'd like to just get this merged so that we can have any documentation at all, right now we have nothing which is an undesirable state of affairs.